src/TestPlan.lua

```diff
@@ -105,7 +105,15 @@ local function newEnvironment(currentNode, extraEnvironment)
 	env.fdescribe = env.describeFOCUS
 	env.xdescribe = env.describeSKIP
 
-	env.expect = Expectation.new
+	env.expect = setmetatable({
+		extend = function(...)
+			error(string.format("Cannot call %q from within a %q node.", "expect.extend", "describe"))
```

Comment:
You can use single quotes for strings containing double quotes. 'Cannot call "expect.extend"...